From 5b1ba8275219946e19c974c3ba3771b519e89db6 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Sun, 28 Feb 2021 18:09:05 +0200
Subject: [PATCH 4/5] imx8mm: remove the UART restrictions

Only D0 have access to UARTs 2 & 4.
Remove the restrictions from UARTs 2 & 4.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c b/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
index 6f98f7304..54e8a7579 100644
--- a/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
@@ -58,8 +58,6 @@ static const struct imx_rdc_cfg rdc[] = {
 	RDC_MDAn(RDC_MDA_M4, DID1),
 
 	/* peripherals domain permission */
-	RDC_PDAPn(RDC_PDAP_UART4, D0R | D0W),
-	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W),
 
 	/* memory region */
 
-- 
2.11.0

